src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -1,86 +1,161 @@
-                <option aria-selected={false}>Other Keyword</option>
-            </>
-        }
-        searchSubmitHandler={action("searched")}
-        selectChangeHandler={action("selectChanged")}
-        selectBlurHandler={action("selectChanged")}
-    />
-);
+const optionsGroup = {
+    Art: [
+        "Academic Painting",
+        "Amazing Painting",
+        "Ancient Statue",
+        "Basic Painting",
+        "Beautiful Statue",
+    ],
+    Bushes: [
+        "Blue-Hydrangea Bush",
+        "Blue-Hydrangea Start",
+        "Holly Bush",
+        "Orange-Tea-Olive Bush",
+        "Pink Camellia Bush",
+    ],
+    Clothing: ["3D Glasses", "Academy Uniform", "Acid-Washed Jacket"],
+    "DIY Recipes": ["Acorn Pochette", "Acoustic Guitar", "Angled Signpost"],
+    Flowers: [
+        "Black Cosmos",
+        "Blue Pansies",
+        "Gold Roses",
+        "Lily of the Valley",
+        "Orange Hyacinth",
+    ],
+    Fossils: [
+        "Acanthostega",
+        "Amber",
+        "Ammonite",
+        "Ankylo Skull",
+        "Anomalocaris",
+        "Australopith",
+    ],
+    Fruits: ["Apples", "Oranges", "Pears", "Coconuts", "Cherries"],
+    Furniture: [
+        "Abstract Wall",
+        "Accessories Stand",
+        "ACNH Switch",
+        "Air Circulator",
+    ],
+    Materials: [
+        "Acorn",
+        "Aquaris Fragment",
+        "Bamboo Piece",
+        "Boot",
+        "Clump of Weeds",
+    ],
+    NPC: ["Celeste", "Kicks", "Leif", "Redd", "Saharah"],
+    Songs: ["Agent K.K.", "Café K.K.", "Comrade K.K."],
+    Tools: [
+        "Axe",
+        "Bamboo Wand",
+        "Fishing Rod",
+        "Shovel",
+        "Slingshot",
+        "Watering Can",
+    ],
+    Villagers: ["Agnes", "Alfonso", "Anchovy", "Axel", "Aurora"],
+};
 
-export const searchWithNoParams = () => (
-    <SearchBar
-        searchBarId="searchBarId"
-        buttonId="searchButtonId"
-        searchBarAriaLabel="search"
-        textFieldAriaLabel="Text Field aria-label"
-        searchSubmitHandler={action("searched")}
-    />
-);
+let showSelect, showHelperText, formErrored, formDisabled;
 
-export const searchWithFormError = () => (
-    <SearchBar
-        searchBarId="searchBarId"
-        searchBarAriaLabel="search"
-        buttonId="searchButtonId"
-        textFieldAriaLabel="Text Field aria-label"
-        searchSubmitHandler={action("searched")}
-        hasError={true}
-        errorMessage={"Error text"}
-    />
+export const searchBar = () => (
+    <>
+        {boolean("Form Errored", false)
+            ? (formErrored = true)
+            : (formErrored = false)}
+        {boolean("Form Disabled", false)
+            ? (formDisabled = true)
+            : (formDisabled = false)}
+        {boolean("Show Select", true)
+            ? (showSelect = true)
+            : (showSelect = false)}
+        {boolean("Show Helper Text", true)
+            ? (showHelperText = true)
+            : (showHelperText = false)}
+        <SearchBar
```

Comment:
![Screen Shot 2020-07-14 at 6 47 50 PM](https://user-images.githubusercontent.com/3579472/87483781-95949b80-c602-11ea-9e72-b10c13c04172.png)

This looks off